portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -77,11 +78,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       datastore.put(taskEntity);
     } else if (maxString != null) {
       try {
-        max = Integer.parseInt(maxString);
-
+        maxNumberOfComments = Integer.parseInt(maxString);
       } catch (NumberFormatException e) {
-        System.out.println("Could not convert to int: " + maxString);
-        max = 1;
+        System.out.println("Could not convert to int|" + maxString+"|");
```

Comment:
Please use `String.format` instead of these string concat chains.

```java
System.out.println(String.format("Could not convert to int: \"%s\".", maxString);
```

Reply:
Moreover I'm going to cut you a bug to switch to a real logger.